py/src/ai/h2o/sparkling/ExternalBackendConf.py

```diff
@@ -187,3 +190,7 @@ def setExternalHadoopExecutable(self, executable):
     def setExternalExtraJars(self, paths):
         self._jconf.setExternalExtraJars(paths)
         return self
+
```

Comment:
I know the PR is still marked as WIP, but wanted to mention this before I forget. Would be good to expose it on H2OConf.R as well

Reply:
Will do, thank!